packages/cadl-ranch-specs/http/azure/resource-manager/models/resources/location.tsp

```diff
@@ -0,0 +1,179 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using Azure.Core;
+using Azure.ResourceManager;
+using TypeSpec.OpenAPI;
+
+namespace Azure.ResourceManager.Models.Resources;
+
+@resource("locationTrackedResources")
+@parentResource(ResourceGroupLocationResource)
+model LocationTrackedResource is TrackedResource<LocationTrackedResourceProperties> {
+  ...ResourceNameParameter<LocationTrackedResource>;
+}
+
+/** Employee properties */
+model LocationTrackedResourceProperties {
+  @doc("The description of the resource.")
+  description?: string;
+
+  /** The status of the last operation. */
+  @visibility("read")
+  provisioningState?: ProvisioningState;
+}
+
+@armResourceOperations
+interface LocationTrackedResources {
+  @scenario
+  @scenarioDoc("""
+    Resource GET operation.
+    Expected path: /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Azure.ResourceManager.Models.Resources/locations/eastus/locationTrackedResources/location
```

Comment:
the resource name `location` is ambiguous

Reply:
The resource name has been changed form `location` to `resource`. The new version commit https://github.com/Azure/cadl-ranch/pull/703/commits/ba81c2928d44a2826d77ed191839a89b1133b39d